Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityDeviceManager.cs

```diff
@@ -95,7 +101,7 @@ private WindowsMixedRealityController GetOrAddController(InteractionSourceState
             Handedness controllingHand;
             switch (interactionSourceState.source.handedness)
             {
-                case InteractionSourceHandedness.Unknown:
+                default:
```

Comment:
feels weird to have default come first, but don't change it because of me :)

Reply:
I don't mind reverting, or moving this to the bottom, if it makes more sense.